src/app/util/util.ts

```diff
@@ -1,26 +1,5 @@
-import { ComponentFixture } from '@angular/core/testing';
-import { By } from '@angular/platform-browser';
-import { NavigationEnd, Router } from '@angular/router';
+export const getFormattedCurrentDate = () => new Date().toISOString().split('T')[0];
 
-import { BehaviorSubject } from 'rxjs';
+export const isValueNotIntegerNumber = (value: any) => !Number.isInteger(value);
```

Comment:
It might be better to check if `isInteger(value)` and then add `!` where it is necessary to check if it is not an integer...

Reply:
Ok